installer/roles/kubernetes/templates/deployment.yml.j2

```diff
@@ -269,12 +269,44 @@ spec:
               containerPort: 5672
           livenessProbe:
             exec:
-              command: ["rabbitmqctl", "status"]
+              command:
+              - /usr/bin/python
+              - -c
+              - |
+                import httplib
```

Comment:
This assumes you have python3, which may not be a safe assumption to make downstream cc @shanemcd

Reply:
```python
try:
    from http.client import HTTPConnection
except ImportError
    from httplib import HTTPConnection
```

Reply:
I've modified the import according to your example. Thanks for spotting it.